time timestamp

float32 acceleration
float32 steering
